src/background/NotificationWindow/createBrowserWindow.ts

```diff
@@ -1,25 +1,16 @@
 import type { Windows } from 'webextension-polyfill';
 import browser from 'webextension-polyfill';
 import { nanoid } from 'nanoid';
+import { getPopupUrl } from 'src/ui/shared/getPopupUrl';
+import { setUrlContext } from 'src/ui/shared/setUrlContext';
```

Comment:
I think both of these belong to `/src/shared/`

Reply:
✅ moved